watson-developer-cloud/discovery-components
# feat: add DocumentPreview error message

#### What do these changes do/fix?
Adds an error message to document preview for when JSON and CSV files do not have a passage to load. 

https://github.ibm.com/Watson-Discovery/the-tooling-core-tracker/issues/1192

**Before for CSV:**
<img width="1157" alt="Screen Shot 2020-03-04 at 4 46 29 PM" src="https://user-images.githubusercontent.com/59846843/75930218-bb7d8880-5e37-11ea-9305-7d347e6bf9d8.png">

**After for CSV and JSON:**
<img width="1196" alt="Screen Shot 2020-03-05 at 12 39 11 PM" src="https://user-images.githubusercontent.com/59846843/76013995-7eb89c80-5ede-11ea-8b98-31c849fdda5f.png">

#### How do you test/verify these changes?
On Example App:
1. Run the Components Example App against a project with a CSV file and do NOT specify a search result field.
2. See that the error message appears in the Document tab.
3. On the cluster, set a search result field. 
4. On the Example App, see that the error message is gone and the field is shown.

On Cluster:
1. Link components and tooling. Make sure you also link components styles.
2. Look at a project with a CSV file and NOT a specified search result field.
3. See error message on Document tab.
4. Specify a search result field, go back to Document tab and see that the error message is gone.

#### Have you documented your changes (if necessary)?
Tests have been updated to reflect this change.

#### Are there any breaking changes included in this pull request?
No
